script.js

```diff
@@ -0,0 +1,27 @@
+function check(number) {
+    if(number=== "" || isNaN(number)) {
+        alert("Prosze wpisać wartość liczbowa");
+        return false;
+    } else
+    return true;
+}
+function celcjusz()
+{
+	let number=document.getElementById('number').value;
+	let celcjusz=number*'1.8'+'32';
```

Comment:
why do you use strings `'1.8'` etc instead of numeric number `1.8`?
It looks hacky

Reply:
done